app/kafka/manager/KafkaManagerActor.scala

```diff
@@ -172,6 +187,8 @@ object ClusterConfig {
             curatorConfig,
             enabled,version,
             jmxEnabled.getOrElse(false),
+            jmxUser.getOrElse(Some("")),
```

Comment:
if I remove it, same for the other explicit default value declarations, I do get an error on compilation regarding the signatures at: "case class ClusterConfig()" and "object ClusterConfig [...] def apply()"